ciri/conf/config.h

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2018 IOTA Stiftung
+ * https://github.com/iotaledger/entangled
+ *
+ * Refer to the LICENSE file for licensing information
+ */
+
+#ifndef __COMMON_CONFIGURATION__
```

Comment:
Please follow the scoping pattern for include guards.
`__CIRI_CONF_CONFIG_H__` for example.